### PR TITLE
Static images calibration

### PR DESCRIPTION
**Pull request by macmacal**
Extending your example, I've prepared scripts for working with directories of static images. 
Please review this PR before proceeding.

I also suggest to rename the repository to `camera_calibration_scripts`. Firstly, to stick with the snake_case convention. Secondly, to improve the self-explanatory. (You will need to change the remotes on your machines with the `git remote set-url` command).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+### Process a dataset
+Edit the `image_calibration/process_dataset.sh` file to your needs.
+```bash
+./image_calibration/process_dataset.sh "path/to/dataset/images/folder"
+```
+----
+
+## Requirements
+```
+numpy
```
Comment (NieTrawisz) [inline README.md]: Isn't numpy a requirement for opencv?

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+opencv-python
+numpy
```
Comment (NieTrawisz) [inline requirements.txt]: Isn't numpy a requirement for opencv?

**Review by NieTrawisz — COMMENTED** [review]
I think its ok but we can remove numpy from requirements since it is already a requirement for opencv

File: README.md (hunk 1 above)
Comment (macmacal) [reply README.md]: Yep it is. I removed this duplicated requirement.

File: requirements.txt (hunk 2 above)
Comment (macmacal) [reply requirements.txt]: Yep it is. I removed this duplicated requirement.

**Comment by macmacal** [comment]
@NieTrawisz Done. I'm waiting for your merge approval.

**Review by NieTrawisz — APPROVED** [review]
Looks Good To Me

**Pull request merged**